README.md

```diff
@@ -224,11 +224,21 @@ To build both parts, run the following commands:
 
 ```sh
 $ just rust
-$ just go
+$ just build
 ```
 
 To build the Go part, run:
 
+```sh
+$ just build
+```
+
+To run tests, run:
```

Comment:
The following Testing Section describes how to test the project. So I would skip this part here, and I would only keep the `just go` -> `just build` modification.

Does it work for you?

Reply:
Yes, I will update to a minimal change